### PR TITLE
Make definitions consistent with the rest, and EDF doesn't like floats

### PR DESCRIPTION
**Pull request by stephenwhitmarsh**

### CODE REVIEW

File: module/recordsignal/recordsignal.py [hunk 1]
```diff
@@ -91,10 +91,10 @@ def _start():
     # this can be used to show parameters that have changed
     monitor = EEGsynth.monitor(name=name, debug=patch.getint('general', 'debug'))
 
-    MININT16 = -0xffff / 2 - 1
-    MAXINT16 = 0xffff / 2 - 1
-    MININT32 = -0xffffffff / 2 - 1
-    MAXINT32 = 0xffffffff / 2 - 1
+    MININT16 = -np.power(2, 15)
```
Comment (robertoostenveld) [inline module/recordsignal/recordsignal.py:94]: can you make `2` and `2.` consistent over the four?

Comment (robertoostenveld) [reply module/recordsignal/recordsignal.py:94]: other than that it is fine, you can merge it yourself if you make the change

**Comment by stephenwhitmarsh** [comment]
everywhere they are used as floats, except in this module, for EDF (for WAV
it uses floats).
Q: the

**Comment by stephenwhitmarsh** [comment]
silly Q, but;
a) all integers in all the code and risk issues?
b) only integers in recordsignal, but for both 16 and 32 bit range?
c) keep it like this?

**Comment by robertoostenveld** [comment]
EDF uses int16 values in the files that - when reading in the file - are multiplied with a calibration value that scales and converts the integer to a float. Other than that Python usually does not care (like MATLAB) about the difference between teh two. There are some cases where it matters, but you are not likely to encounter them.  

I think the maxint32 variables are there just for completeness and reference, but not used.

**Comment by robertoostenveld** [comment]
What is actually the reason that you are changing this code? Did you encounter a specific issue? Your questions make me think that it might be better not to touch this code.

**Pull request merged**